lua/telescope/config.lua

```diff
@@ -113,6 +113,8 @@ function config.set_defaults(defaults)
   set("border", {})
   set("borderchars", { '─', '│', '─', '│', '╭', '╮', '╯', '╰'})
 
+  set("path_display", {})
```

Comment:
This needs a description here explaining how to use it -- otherwise people won't know what to do.

Also, we should make sure that every reference to `shorten_path`, including in the README is removed.

Reply:
I added [this wiki page](https://github.com/nvim-telescope/telescope.nvim/wiki/Path-Display-Configuration). Feel free to edit anyone.

I'm still working on the PR, but plan on linking this from the README, since this setting is too much to be explained in one line.

Reply:
> I added [this wiki page](https://github.com/nvim-telescope/telescope.nvim/wiki/Path-Display-Configuration). Feel free to edit anyone.
> 
> I'm still working on the PR, but plan on linking this from the README, since this setting is too much to be explained in one line.

😍 😍